src/components/controls/GameControls.jsx

```
@@ -0,0 +1,22 @@
+import React from 'react';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import ControlsButton from './ControlsButton';
+
+function GameControls() {
+  return (
+    <div class="game-controls" >
```

ESLint

[error] Unexpected usage of doublequote.

(@stylistic/jsx-quotes)